assets/airports/eddl.json

```diff
@@ -0,0 +1,3131 @@
+            "entryPoints": {
+                "LONSO": [["LONSO", "A70+"]]
+            },
+            "body": ["DL456", ["REGNO", "A30+"]],
+            "rwy": {
+                "EDDL23R": []
+            },
+            "draw": [
+                ["LONSO", "DL456"]
+            ]
+        },
+        "LONSO1B": {
+            "icao": "LONSO1B",
+            "name": "Lonso One Bravo",
+            "entryPoints": {
+                "LONSO": [["LONSO", "A70+"]]
+            },
+            "body": ["DL455", ["METMA", "A30+"]],
+            "rwy": {
+                "EDDL23L": []
+            },
+            "draw": [
+                ["LONSO", "DL455"]
+            ]
+        },
+        "NISOX1A": {
+            "icao": "NISOX1A",
+            "name": "Nisox One Alpha",
+            "entryPoints": {
+                "NISOX": [["NISOX", "A70+"]]
+            },
+            "body": ["DL456", ["REGNO", "A30+"]],
+            "rwy": {
+                "EDDL23R": []
+            },
+            "draw": [
+                ["NISOX", "DL456"]
+            ]
+        },
+        "NISOX1B": {
+            "icao": "NISOX1B",
+            "name": "Nisox One Bravo",
+            "entryPoints": {
+                "NISOX": [["NISOX", "A70+"]]
+            },
+            "body": ["DL455", ["METMA", "A30+"]],
+            "rwy": {
+                "EDDL23L": []
+            },
+            "draw": [
+                ["NISOX", "DL455"]
+            ]
+        },
+        "OBOMA1A": {
+            "icao": "OBOMA1A",
+            "name": "Oboma One Alpha",
+            "entryPoints": {
+                "OBOMA": [["OBOMA", "A70+"]]
+            },
+            "body": ["DL456", ["REGNO", "A30+"]],
+            "rwy": {
+                "EDDL23R": []
+            },
+            "draw": [
+                ["OBOMA", "DL456"]
+            ]
+        },
+        "OBOMA1B": {
+            "icao": "OBOMA1B",
+            "name": "Oboma One Bravo",
+            "entryPoints": {
+                "OBOMA": [["OBOMA", "A70+"]]
+            },
+            "body": ["DL455", ["METMA", "A30+"]],
+            "rwy": {
+                "EDDL23L": []
+            },
+            "draw": [
+                ["OBOMA", "DL455"]
+            ]
+        },
+        "RUGAP1A": {
+            "icao": "RUGAP1A",
+            "name": "Rugap One Alpha",
+            "entryPoints": {
+                "RUGAP": [["RUGAP", "A70+"]]
+            },
+            "body": ["DL456", ["REGNO", "A30+"]],
+            "rwy": {
+                "EDDL23R": []
+            },
+            "draw": [
+                ["RUGAP", "DL456"]
+            ]
+        },
+        "RUGAP1B": {
+            "icao": "RUGAP1B",
+            "name": "Rugap One Bravo",
+            "entryPoints": {
+                "RUGAP": [["RUGAP", "A70+"]]
+            },
+            "body": ["DL455", ["METMA", "A30+"]],
+            "rwy": {
+                "EDDL23L": []
+            },
+            "dr
```

Comment:
Line 1965:
No altitude restriction at XAMOD for TEBRO23

Reply:
Same thing about source:
![image](https://user-images.githubusercontent.com/19408472/42474029-426190fe-83be-11e8-928a-60f190e42ddc.png)
![image](https://user-images.githubusercontent.com/19408472/42474055-4ea4e87a-83be-11e8-92a8-5d7a34dd30e7.png)